src/olympia/reviewers/views.py

```diff
@@ -1140,7 +1140,7 @@ def download_git_stored_file(request, version_id, filename):
         raise http.Http404
 
     if version.channel == amo.RELEASE_CHANNEL_LISTED:
-        is_owner = acl.check_addon_ownership(request, addon, dev=True)
+        is_owner = acl.check_addon_ownership(request, addon, allow_developer=True)
```

Comment:
do we want to allow_site_permission here too?

Reply:
I don't think so, since they'll be auto-generated it's not really useful to look at them in code manager (I'm not even convinced we need to allow add-on developers to access the code manager APIs, btw...)